site/en/tutorials/structured_data/imbalanced_data.ipynb

```diff
@@ -275,6 +274,31 @@
         "id": "8a_Z_kBmr7Oh"
       },
       "source": [
+        "We check whether the distribution of the classes in the three sets is about the same or not."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "96520cffee66"
+      },
+      "outputs": [],
+      "source": [
+        "print(f'Average class probability in training set:   {train_labels.mean():.4f}')\n",
+        "print(f'Average class probability in validation set: {val_labels.mean():.4f}')\n",
+        "print(f'Average class probability in test set:       {test_labels.mean():.4f}')"
+      ]
+    },
+    {
+      "attachments": {},
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "8a_Z_kBmr7Oh"
+      },
+      "source": [
+        "Given the small number of positive labels, this seems about right. In case the average class probability deviates by a large amount, one could use a stratified splitting scheme.\n",
```

Comment:
```suggestion
        "Given the small number of positive labels, this seems about right.\n",
```

I try to avoid mentioning new techniques without more info or a link.